batch_import/README.md

```diff
@@ -244,49 +243,61 @@ Which will output the following:
 
 ```sh
 $ python download_labels.py --help
-usage: download_labels.py [-h] [-a ACCESS_TOKEN] [-c] [-x] [-t] [-v] [-q]
-                         [job_id] [output_file]
+usage: download_labels.py [-h] [-o OUTPUT_FILE] [-a ACCESS_TOKEN]
+                          [-w LABEL_WHITE_LIST] [-f LABEL_WHITE_LIST_FILE]
+                          [-c] [-x] [-t] [-v] [-q]
+                          [job_id]
 
-This script accepts a Camio `job_id`, finds the time-boundaries and cameras 
-associated with that job, and iterates over that time range while downloading all 
-of the labels that Camio has assigned to those events.
+This script will take a Camio job_id, find the time-boundaries and cameras involved in that job, and iterate over that
+time range while downloading all of the labels that Camio has annotated the events with.
```

Comment:
@jhallard please use the language I previously suggested before this change:

This script accepts a Camio `job_id`, finds the time-boundaries and cameras associated with that job, and iterates over that time range while downloading all of the labels that Camio has assigned to those events.